### ISSUE TITLE
util-linux fails to install

### ISSUE THREAD

**Issue by mikeandmore** [Linuxbrew/homebrew-core#1523]
```
mike@sea:(master)% brew upgrade util-linux
==> Upgrading 1 outdated package, with result:
util-linux 2.29
==> Upgrading util-linux
==> Downloading https://www.kernel.org/pub/linux/utils/util-linux/v2.29/util-linux-2.29.tar.xz
Already downloaded: /home/mike/.cache/Homebrew/util-linux-2.29.tar.xz
==> ./configure --disable-silent-rules --prefix=/home/mike/.linuxbrew/Cellar/util-linux/2.29 --disable-use-tty-group --disable-kill --disable-cal --disable-ul
==> make install
Last 15 lines from /home/mike/.cache/Homebrew/Logs/util-linux/02.make:
     during execution
   - add LIBDIR to the 'LD_RUN_PATH' environment variable
     during linking
   - use the '-Wl,-rpath -Wl,LIBDIR' linker flag
   - have your system administrator add LIBDIR to '/etc/ld.so.conf'

See any operating system documentation about shared libraries for
more information, such as the ld(1) and ld.so(8) manual pages.
----------------------------------------------------------------------
make[3]: Leaving directory `/tmp/util-linux-20170107-2205-xdbaor/util-linux-2.29'
make[2]: *** [install-am] Error 2
make[2]: Leaving directory `/tmp/util-linux-20170107-2205-xdbaor/util-linux-2.29'
make[1]: *** [install-recursive] Error 1
make[1]: Leaving directory `/tmp/util-linux-20170107-2205-xdbaor/util-linux-2.29'
make: *** [install] Error 2

READ THIS: https://github.com/Linuxbrew/brew/blob/master/docs/Troubleshooting.md#troubleshooting

````

It seems util-linux is trying to install into system systemd unit directories?

```
 /usr/bin/install -c -m 644 sys-utils/fstrim.service sys-utils/fstrim.timer misc-utils/uuidd.service misc-utils/uuidd.socket '/usr/lib/systemd/system'
/usr/bin/install: cannot remove ‘/usr/lib/systemd/system/fstrim.service’: Permission denied
/usr/bin/install: cannot remove ‘/usr/lib/systemd/system/fstrim.timer’: Permission denied
/usr/bin/install: cannot create regular file ‘/usr/lib/systemd/system/uuidd.service’: Permission denied
/usr/bin/install: cannot create regular file ‘/usr/lib/systemd/system/uuidd.socket’: Permission denied
make[3]:  /usr/bin/mkdir -p '/home/mike/.linuxbrew/Cellar/util-linux/2.29/include/uuid'
*** [install-systemdsystemunitDATA] Error 1
```

**Comment by maxim-belkin** [comment]
please share gist-logs.

**Comment by mikeandmore** [comment]
gist log https://gist.github.com/a98eea0ddcf8d8e1e874cc066265cca8

**Comment by sjackman** [comment]
Fixed by #1524.
